modules/simple_forms_api/app/form_mappings/vba_21_4140.json.erb

```diff
@@ -35,18 +40,21 @@
   <%# 8. Alternate Telephone Number %>
   "F[0].Page_1[0].AlternateTelephoneNumber[0]": "<%= form.phone_alternate %>",
 
-  "F[0].Page_1[0].CurrentMailingAddress_ZIPOrPostalCode_LastFourNumbers[0]": "<%= form.data.dig('CurrentMailingAddress_ZIPOrPostalCode_LastFourNumbers') %>",
-  "F[0].Page_1[0].CurrentMailingAddress_ZIPOrPostalCode_FirstFiveNumbers[0]": "<%= form.data.dig('CurrentMailingAddress_ZIPOrPostalCode_FirstFiveNumbers') %>",
-  "F[0].Page_1[0].CurrentMailingAddress_Country[0]": "<%= form.data.dig('CurrentMailingAddress_Country') %>",
-  "F[0].Page_1[0].CurrentMailingAddress_StateOrProvince[0]": "<%= form.data.dig('CurrentMailingAddress_StateOrProvince') %>",
-  "F[0].Page_1[0].CurrentMailingAddress_City[0]": "<%= form.data.dig('CurrentMailingAddress_City') %>",
-  "F[0].Page_1[0].CurrentMailingAddress_ApartmentOrUnitNumber[0]": "<%= form.data.dig('CurrentMailingAddress_ApartmentOrUnitNumber') %>",
-  "F[0].Page_1[0].CurrentMailingAddress_NumberAndStreet[0]": "<%= form.data.dig('CurrentMailingAddress_NumberAndStreet') %>",
+  <%# 9. Current Mailing Address of Veteran %>
+  "F[0].Page_1[0].CurrentMailingAddress_NumberAndStreet[0]": "<%= form.address.address_line1 %>",
+  "F[0].Page_1[0].CurrentMailingAddress_ApartmentOrUnitNumber[0]": "<%= form.address.address_line2 %>",
+  "F[0].Page_1[0].CurrentMailingAddress_City[0]": "<%= form.address.city %>",
+  "F[0].Page_1[0].CurrentMailingAddress_StateOrProvince[0]": "<%= form.address.state_code %>",
+  "F[0].Page_1[0].CurrentMailingAddress_Country[0]": "<%= form.address.country_code_iso2 %>",
```

Comment:
There is only room on the PDF for two characters and truncating an ISO3 code would produce invalid country codes.